src/main/java/org/example/objects/UserList.java

```diff
@@ -0,0 +1,16 @@
+package org.example.objects;
```

Comment:
@SebasBarra02 As I mentioned on MusicList.java, please review that.

Reply:
test cases were added.